src/components/ScrollArea.tsx

```diff
@@ -9,13 +11,24 @@ export type Props = {
   >["orientation"];
 };
 
-export default function ScrollArea({ children, orientation }: Props) {
+function ScrollArea({ children, orientation }: Props) {
+  const ref = useRef<HTMLDivElement>(null);
+
+  useScroller(ref, {
+    orientation: orientation ?? "vertical",
+    // skip emulation since this is the default behavior
+    skip: orientation === "vertical",
```

Comment:
I think a vertical scroller will never make sense. Instead, let's use a `enabled` prop to replace both orientation and skip

Reply:
I have applied your suggestions and renamed the hook to `useXScroller` since this scrolls in x-axis